### ISSUE TITLE
Namespacing for Exception folder isn't quite right.

### ISSUE THREAD

**Issue by BrandonSurowiec** [stymiee/authnetjson#11]
Changing namespaces was a big BC change and should have been done under a 5.0 release. Anyway, the new namespace isn't quite PSR-0/4 compliant. It should be:

```
"autoload": {
    "psr-4": {
        "Authnetjson\\": "src/Authnetjson/"
    }
},
````

And the namespace for the Exceptions folder should be:

```
namespace Authnetjson/Exception;
```

![image](https://user-images.githubusercontent.com/5625680/91717867-50034200-eb60-11ea-83b2-b05d5b3c108f.png)

If you don't get to it, I'll try to make a PR by the end of the week.

**Comment by stymiee** [comment]
@BrandonSurowiec You're right. I totally botched this somehow. I am making updates today and will have an updated release out soon. Thank you for reporting this. The PHP72 branch shows my progress.

**Comment by BrandonSurowiec** [comment]
Not a problem, really appreciate this library!

**Comment by stymiee** [comment]
I just released v4.1.3 to address this issue. Thank you again for reporting it.
